### DISCUSSION TITLE
don't show "static sites" in RSS Feed

### DISCUSSION THREAD

**Discussion by ghost** [adityatelange/hugo-PaperMod#774] [Q&A]
is there a possibility to stop listing "no post sites" in the rss feed?

let me give you an example based on your blog.
In your rss feed (https://adityatelange.in/index.xml) there are also sites listed like "About", "Archive" and all the other pages that are not real articles.

**Comment by adityatelange** [comment]
https://github.com/adityatelange/hugo-PaperMod/blob/530044033e8a9d87d5297b936e37cbfb2b5562e3/layouts/_default/rss.xml#L35

Archive won't be present.
You can hide any post you want. Pls search docs for the front matter variable.

**Reply by ghost** [reply]
sorry, maybe i am blind. I can not find any informations about "front matter variable" in docs. 
Can you link this information to the discussion?
